frontend/ui/widget/inputdialog.lua

```diff
@@ -754,18 +790,21 @@ function InputDialog:_addScrollButtons(nav_bar)
             table.insert(row, {
                 text = _("Find"),
                 callback = function()
+                    local keyboard_hidden_state = not self.keyboard_hidden
+                    self:toggleKeyboard(true) -- hide text editor keyboard
                     local input_dialog
                     input_dialog = InputDialog:new{
                         title = _("Enter text to search for"),
                         stop_events_propagation = true, -- avoid interactions with upper InputDialog
-                        deny_keyboard_hiding = true,
```

Comment:
Is this still used, and if not, could it still be useful in other contexts ? If not, may remove its support completely.

Reply:
> Maybe keep it for the future? 

As you wish - you should have a better feeling than me whether this can still be useful in future contexts.